tools/scripts/lib_install/spdlog.sh

```
@@ -0,0 +1,92 @@
+
+# Check if checkinstall is installed
+set +e
+command -v checkinstall
+checkinstall_installed=$?
+set -e
+
+# Install
+install_command_prefix="${privileged_command_prefix}"
+if [ $checkinstall_installed -eq 0 ] ; then
+  # NOTE: We use "1:${version}" to override the version installed by Ubuntu 18.04
+  install_command_prefix="${install_command_prefix} checkinstall --pkgname '${package_name}' --pkgversion '1:${version}' --provides '${package_name}' --nodoc -y --pakdir \"${deb_output_dir}\""
```

Shellcheck

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)